server/tests/updateOpenTrip.test.js

```diff
@@ -0,0 +1,224 @@
+import chai, { expect } from 'chai';
+import { describe, it } from 'mocha';
+import chaiHttp from 'chai-http';
+import dotenv from 'dotenv';
+import app from '../app';
+import updateData from './fixtures/updateOpenTripData';
+
+dotenv.config();
+
+chai.should();
+chai.use(chaiHttp);
+
+describe('===== Test Update Open Trip Request =====', () => {
+  const token = process.env.PASSING_TOKEN;
+  it('Should update the open one way trip request ', done => {
+    chai
+      .request(app)
+      .post('/api/v1/trips/2')
+      .set('Authorization', token)
+      .send(updateData[0])
+      .end((err, res) => {
+        expect(res.statusCode).to.equal(200);
+        expect(res.type).to.equal('application/json');
+        expect(res.body).to.have.property('status');
+        expect(res.body).to.have.property('message');
+        expect(res.body).to.have.property('data');
+        
+        expect(res).to.have.status(200);
+        expect(res.body.message).to.equal('The one-way-trip details were successfully updated');
```

Comment:
when testing APIs with 200 or 201, you can go one step further and make sure that the data (values of the response properties) are actually what have been sent or expected to be returned. In that case, the tests are more robust as they test contents both key & values of the response object.